File: 2023/domaci/02-trivialni-rovnice/volfmatous/main.cs
using System;

internal class Program
{
    static void Main(string[] args)
    {
        double a = double.Parse(Console.ReadLine());
        double b = double.Parse(Console.ReadLine());
        double c = double.Parse(Console.ReadLine());

        string output;
        if (b == 1)
        {
            output = "undefined";
        }
        else
        {
            double x = (c - a) / (b + 1);
            output = x.ToString();
        }

        Console.WriteLine(output);
    }
}

File: 2023/domaci/02-trivialni-rovnice/volfmatous/main.cpp
#include <iostream>
#include <string>

int main() {
    double a, b, c;
    std::cin >> a >> b >> c;

    std::string output;
    if (b == 1) {
        output = "undefined";
    } else {
        double x = (c - a) / (b + 1);
        output = std::to_string(x);
        output.erase(output.find_last_not_of('0') + 1, std::string::npos);
        output.erase(output.find_last_not_of('.') + 1, std::string::npos);
    }

    std::cout << output << std::endl;

    return 0;
}
